client/components/token-field/index.jsx

```diff
@@ -24,12 +25,27 @@ var TokenField = React.createClass( {
 	propTypes: {
 		suggestions: React.PropTypes.array,
 		maxSuggestions: React.PropTypes.number,
-		value: React.PropTypes.array,
 		displayTransform: React.PropTypes.func,
 		saveTransform: React.PropTypes.func,
 		onChange: React.PropTypes.func,
 		tokenStatus: React.PropTypes.func,
-		isBorderless: React.PropTypes.bool
+		isBorderless: React.PropTypes.bool,
+		value: function( props ) {
```

Comment:
Well, good news! I tested this out in production by adding a `console.log()` in the `value` propType. I then tested in `development` and got the logging, but when I tested with `CALYPO_ENV=production make run`, I didn't get logging. Which leads me to believe that propTypes don't run in production.